recipes-samples/images/rpb-console-image-lava.bb

```diff
@@ -0,0 +1,5 @@
+require rpb-console-image.bb
+
+CORE_IMAGE_BASE_INSTALL += " \
+    linaro-lava-tests \
```

Comment:
Right, I mistakenly assumed that 'linaro-lava-tests' was the packagegroup with all the needed test bits.